nugs-pr1-fixes.md

```
@@ -0,0 +1,170 @@
+SchedulerService named hasScheduleColumn(table, column string) that runs a
+PRAGMA table_info query (SELECT COUNT(*) FROM pragma_table_info('<table>') WHERE
+name = '<column>'), scans the result into an int, returns false on any
+query/scan error, and returns true only if the count is greater than zero; place
+this function outside the reviewed ranges in the same file and ensure it uses
+s.DB for the query and appropriate return semantics.
+
+---
+
+- [x] [COMMITTABLE SUGGESTION - coderabbitai[bot] - cron/internal/services/scheduler.go:682]
+// Derive type from parameters (if present) when column is absent
+		if !hasTypeColumn {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)